### PR TITLE
Re-change the options for rebuild index batch number

### PR DESCRIPTION
**Pull request by bright-starry-sky**
I'm so sorry. the configuration file was wrong in the previous PR #540. Let's correct that in this PR.
:sweat_smile:

### CODE REVIEW

**Review by randomJoe211 — DISMISSED** [review]

**Comment by critical27** [comment]
Hold on for one sec.

**Comment by bright-starry-sky** [comment]
I discussed it with @critical27 , `rebuild_index_batch_num` is not very effective as the speed limit. added the new option `rebuild_index_process_interval` for speed limit.

File: src/storage/StorageFlags.cpp [hunk 1]
```diff
@@ -26,6 +26,9 @@ DEFINE_int32(rebuild_index_batch_num, 256,
 DEFINE_int32(rebuild_index_locked_threshold, 256,
              "The locked threshold will refuse writing.");
 
+DEFINE_int32(rebuild_index_process_interval, 30000,
```
Comment (critical27) [inline src/storage/StorageFlags.cpp:29]: 3ms is enough I think? 30ms means there are 33 request at most for one second.

Comment (bright-starry-sky) [reply src/storage/StorageFlags.cpp]: > 3ms is enough I think? 30ms means there are 33 request at most for one second.

good catch. fixed.

**Review by critical27 — APPROVED** [review]
LGTM

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**